sovrin_node/server/pool_req_handler.py

```diff
@@ -0,0 +1,64 @@
+from copy import deepcopy
+
+from ledger.serializers.json_serializer import JsonSerializer
+from plenum.common.constants import TARGET_NYM, DATA, ALIAS, SERVICES
+
+from plenum.common.ledger import Ledger
+from plenum.persistence.pruning_state import PruningState
+from plenum.server.pool_req_handler import PoolRequestHandler as PHandler
+from sovrin_common.auth import Authoriser
+from sovrin_common.constants import NODE
+from sovrin_node.persistence.idr_cache import IdrCache
+
+
+class PoolRequestHandler(PHandler):
```

Comment:
We should avoid leaking details of base classes, its a request handler and in most cases we should not bother the devs with details like its base class, also people dont need to use Plenum's pool request handler unless they are extending it